### PR TITLE
Add configuration to requests instrumentation readme

### PR DESCRIPTION
**Pull request by lmolkova**
# Description
Added steps to enable requests instrumentation (currently one has to look to integration tests to see how it has to be done)

## Type of change

- [x] This change requires a documentation update

# How Has This Been Tested?

created sample app and saw requests instrumentation tracking span

# Does This PR Require a Core Repo Change?

- [x] No.

# Checklist:

See [contributing.md](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/CONTRIBUTING.md) for styleguide, changelog guidelines, and more.

- [x] Followed the style guidelines of this project
- [ ] Changelogs have been updated
- [x] Unit tests have been added
- [x] Documentation has been updated

### CODE REVIEW

**Review by codeboten — APPROVED** [review]
Thanks for the PR!

Normally the docs for this would be available here: https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/requests/requests.html and look something like https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/redis/redis.html. This is not working currently because of #552.

**Comment by lmolkova** [comment]
@codeboten  thanks for the review and the context! Do I have to update changelog to make pipeline happy?

File: instrumentation/opentelemetry-instrumentation-requests/README.rst [hunk 1]
```diff
@@ -15,6 +15,15 @@ Installation
 ::
 
      pip install opentelemetry-instrumentation-requests
+     
+Configuration
+------------
+
+.. code:: python
```
Comment (lonewolf3739) [inline instrumentation/opentelemetry-instrumentation-requests/README.rst:22]:
```suggestion
-------------

.. code-block:: python
```

**Comment by codeboten** [comment]
> @codeboten thanks for the review and the context! Do I have to update changelog to make pipeline happy?

Nope, looks like @lonewolf3739 already added the skip changelog label, should be good to go.

**Comment by lzchen** [comment]
This isn't really consistent with the documentation for the rest of our instrumentations (the `Configuration` sections in [django](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation/opentelemetry-instrumentation-django#configuration) and [flask](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation/opentelemetry-instrumentation-flask#configuration) pertains to actual configuration rather than usage). How to use the instrumentations are shown in the [docstrings](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/__init__.py#L20) instead.

**Review by owais — APPROVED** [review]

**Pull request merged**